features/mogelijkOnjuist.feature

```diff
@@ -315,19 +315,15 @@ Functionaliteit: Mogelijk onjuist
       | Standplaats      | Heeft als nevenadres                     | adresseerbareobjecten | 0301030000017617 | documentdatum       | adressen       | nummeraanduidingIdentificaties | Mogelijk is ten onrechte een nevenadres toegekend of ontbreekt de relatie met een nevenadres. |
       | Ligplaats        | Heeft als nevenadres                     | adresseerbareobjecten | 0301020000024211 | documentdatum       | adressen       | nummeraanduidingIdentificaties | Mogelijk is ten onrechte een nevenadres toegekend of ontbreekt de relatie met een nevenadres. |
 
-    Abstract Scenario: leveren mogelijkOnjuist bij met fields vragen naar link die mogelijk onjuiste identificatie bevat
+    Abstract Scenario: leveren mogelijkOnjuist bij met fields vragen naar link die mogelijk naar een onjuist object verwijst.
       Gegeven in object <Objecttype> is gegeven <Attribuut in de BAG> in onderzoek
       Als ik de resource <Resource> opvraag met fields=_links.<Fields>
       Dan bevat het antwoord property mogelijkOnjuist.<Property> met waarde true
       En bevat het antwoord property mogelijkOnjuist.toelichting met een waarde
 
       Voorbeelden:
       | Objecttype       | <Attribuut in de BAG>                    | <Resource>            | Fields                      | Property                       |
```

Comment:
De < en > tekens moeten weg bij <Attribuut in de BAG> en <Resource>

Reply:
@MelvLee dit (en je volgende opmerking) stonden al zo in de feature. Ik denk dat het handiger is dat we de correcties daarvan nu niet bij Nicole leggen. In deze pull request gaat het om de implementatie die straks naar productie moet. Kan jij apart een pull request indienen om deze correcties door te voeren?